### ISSUE TITLE
Opened to PR?

### ISSUE THREAD

**Issue by Guts** [santalvarez/python-rule-engine#8]
Hello @santalvarez 

Thanks for your project that sounds really promising for me since it relies on the JSON Rules ecosystem.

Is the project actively maintained and accepting PRs? I would like to use it in one of my project and I need some changes:

- [ ] logs messages (especially when a condition match fails)
- [ ] support for https://github.com/CacheControl/json-rules-engine/blob/master/docs/rules.md#condition-reference
- [ ] type hint

I obviously can manage it in a fork but I would like to know if you are welcoming PRs and have time to review them.

**Comment by santalvarez** [comment]
Hi!
Yes the project is actively maintained and I will gladly review a PR.
All the changes you mention sound great, the only thing I would like is to be able to turn the logging on or off.

**Comment by Guts** [comment]
> Hi! Yes the project is actively maintained and I will gladly review a PR. All the changes you mention sound great,

Great!

> the only thing I would like is to be able to turn the logging on or off.

What about the classic behavior: warning by default (including errors and critical), except if the log level is manually decrease to INFO or DEBUG?

**Comment by santalvarez** [comment]
Yeah, for logging when a rule matches or not I would use INFO or DEBUG. Then if you want to add logging when a rule can't be added (because of incorrect format for example) then i would use WARNING like you said.
